wp-includes/class-wp-service-worker-scripts.php

```diff
@@ -176,6 +176,8 @@ public function do_item( $handle, $group = false ) {
 			@_doing_it_wrong( 'WP_Service_Worker_Scripts::register', esc_html( $error ), '0.1' ); // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged, WordPress.PHP.NoSilencedErrors.Discouraged -- We want the error in the PHP log, but not in the JS output.
 			printf( "console.warn( %s );\n", wp_service_worker_json_encode( $error ) ); // phpcs:ignore WordPress.XSS.EscapeOutput, WordPress.Security.EscapeOutput
 		}
+
+		return $invalid;
```

Comment:
```suggestion
		return ! $invalid;
```

The way `WP_Scripts` works, we should return `true` if everything works fine, not the other way around.

Reply:
Thank you for noticing!

Done.